apps/aecore/lib/aecore/naming/tx/name_update_tx.ex

```diff
@@ -49,14 +50,22 @@ defmodule Aecore.Naming.Tx.NameUpdateTx do
 
   # Callbacks
 
+  # TODO integrate Identifiers
```

Comment:
is this todo still to be done?

Reply:
This is just a part of unused comments, will remove them soon.